thirty-three-across/thirty-three-across-htb-validator.js

```diff
@@ -23,15 +23,39 @@ function partnerValidator(configs) {
     var result = Inspector.validate({
         type: 'object',
         properties: {
+            siteId: {
+                type: 'string',
+                minLength: 1
+            },
+            test: {
+                type: 'boolean',
+                optional: true
+            },
             xSlots: {
                 type: 'object',
                 properties: {
                     '*': {
                         type: 'object',
                         properties: {
-                            placementId: {
+                            productId: {
```

Comment:
Also the same here? Should we verify if product options to only be ['siab', 'inview']?